config/Backup/models/production_backup.rb

```diff
@@ -75,6 +75,24 @@
     local.keep       = 45
   end
 
+  ##
+  #  store the backup on AWS S3
+  #
+  store_with S3 do |s3|
+    # AWS Credentials
+    s3.access_key_id     = ENV['SHF_AWS_S3_BACKUP_KEY_ID']
+    s3.secret_access_key = ENV['SHF_AWS_S3_BACKUP_SECRET_ACCESS_KEY']
+    # Or, to use a IAM Profile:
+    # s3.use_iam_profile = true
+
+    s3.region             = ENV['SHF_AWS_S3_BACKUP_REGION']
+    s3.bucket             = ENV['SHF_AWS_S3_BACKUP_BUCKET']
+    s3.path               = ENV['SHF_AWS_S3_BACKUP_PATH']
+
+    s3.encryption = :aes256
+
+  end
+
```

Comment:
Good catch about the db.password.  I'll add it to `.env.example` (and will  try to make sure we've included all ENV vars in there)

Re: how  do the backups happen?     I'm not exactly sure what you're asking.  I'm not sure if you're asking about how they're _kicked off_ or how they're _defined_.  So here's a little about both:

- The [backup gem](https://backup.github.io/backup/v4/) provides the _definition_ of the backup using its DSL.  That's what is in `config/Backup/models/production_backup.rb`  

- The _process_ of backing up is _kicked off_  and scheduled by the whenever gem.  The schedule for that is in our `config/Backup/config/schedule.rb` file.

 [Backups in our wiki with refs that describe the process in more detail.](https://github.com/AgileVentures/shf-project/wiki/Production-server#backups)

Does that cover what you were asking?